### PR TITLE
New version: Zen-Team.Zen-Browser version 1.0.0-a.31

### PR DESCRIPTION
**Pull request by sitiom**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/169939)

### CODE REVIEW

File: manifests/z/Zen-Team/Zen-Browser/1.0.0-a.31/Zen-Team.Zen-Browser.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created with komac v2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Zen-Team.Zen-Browser
+PackageVersion: 1.0.0-a.31
+InstallerType: exe
```
Comment (UnownPlain) [inline manifests/z/Zen-Team/Zen-Browser/1.0.0-a.31/Zen-Team.Zen-Browser.installer.yaml:6]: The package won't install correctly if this isn't set to `nullsoft`
```suggestion
InstallerType: nullsoft
```

**Review by UnownPlain — CHANGES_REQUESTED** [review]

**Comment by sitiom** [comment]
Works:
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-08-28 01:05:09.028 [FAIL] Installer failed security check. Url: https://github.com/zen-browser/desktop/releases/download/1.0.0-a.31/zen.installer-generic.exe Result: 0x80004005

Automated error analysis suggests -1978335187 may mean Installer failed security check. Another script should retry this every 18 hours. 

Automated error analysis suggests 80004005 may mean Resource missing

(Automated response - build 891.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**